db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_06_06_133651) do
+ActiveRecord::Schema[7.0].define(version: 2023_07_27_102149) do
```

Comment:
If the schema change is not relevant to the PR you should remove the edit

Reply:
[f38d173](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/749/commits/f38d1738264e7d3996999db663972aef998d1e54)